src/mapgen.h

```diff
@@ -177,6 +177,7 @@ class Mapgen {
 	s16 findGroundLevel(v2s16 p2d, s16 ymin, s16 ymax);
 	s16 findLiquidSurface(v2s16 p2d, s16 ymin, s16 ymax);
 	void updateHeightmap(v3s16 nmin, v3s16 nmax);
+	bool updateLiquidHelper(int i, const v3s16 &em);
```

Comment:
Probably should be `private`, and should share the `inline` decorator here as well.